cmd/virt-handler/virt-handler.go

```diff
@@ -152,13 +157,19 @@ func (app *virtHandlerApp) AddFlags() {
 	app.AddCommonFlags()
 
 	flag.StringVar(&app.HostOverride, "hostname-override", hostOverride,
-		"Name under which the node is registered in kubernetes, where this virt-handler instance is running on")
+		"Name under which the node is registered in Kubernetes, where this virt-handler instance is running on")
 
 	flag.StringVar(&app.VirtShareDir, "kubevirt-share-dir", virtShareDir,
 		"Shared directory between virt-handler and virt-launcher")
 
 	flag.DurationVar(&app.WatchdogTimeoutDuration, "watchdog-timeout", defaultWatchdogTimeout,
 		"Watchdog file timeout")
+
+	// TODO: the Device Plugin API does not allow for infinitely available (shared) devices
+	// so the current approach is to register an arbitrary number.
+	// This should be deprecated if the API allows for shared resources in the future
+	flag.IntVar(&app.MaxDevices, "max-devices", maxDevices,
```

Comment:
Just hardcode it to e.g. 1000?

Reply:
Outcome of a discussion with @fabiand was to make this configurable. However since the medium term goal would be to get rid of the need for this setting, the notion of making a magic number isn't terribly disagreeable.

There are memory and consumption side effects to making this number too large. I'd suggest the Kubelet default of 110 is a better choice (we can't realistically need more devices than there are available pods anyways).